docker-compose-quick.yml

```diff
@@ -0,0 +1,80 @@
+version: '3.4'
+
+services:
+
+  lpvs:
+    build:
+      context: .
+    image: ghcr.io/samsung/lpvs:latest
+    restart: always
+    ports:
+      - "7896:7896"
+    environment:
+      ## Required if frontend and backend are different
+      #- frontend.main-page.url=http://localhost:3000
+      #- cors.allowed-origin=http://localhost:3000
+      ## Database Configuration
+      - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
+      - spring.datasource.username=root
+      - spring.datasource.password=
+      - spring.jpa.properties.hibernate.default_schema=lpvs
+      ## Github data for fetching code
+      - github.login=
+      - github.token=
+      - github.api.url=https://api.github.com
+      - github.secret=LPVS
+      ## Google OAuth Login
+      - spring.security.oauth2.client.registration.google.client-id=GOOGLE_CLIENT_ID
+      - spring.security.oauth2.client.registration.google.client-secret=GOOGLE_CLIENT_SECRET
+      - spring.security.oauth2.client.registration.google.redirect-uri=http://localhost:7896/login/oauth2/code/google
+      - spring.security.oauth2.client.registration.google.scope=profile, email
+      ## Github OAuth Login
+      - spring.security.oauth2.client.registration.github.client-id=GITHUB_CLIENT_ID
+      - spring.security.oauth2.client.registration.github.client-secret=GITHUB_CLIENT_SECRET
+      - spring.security.oauth2.client.registration.github.redirect-uri=http://localhost:7896/login/oauth2/code/github
+      - spring.security.oauth2.client.registration.github.scope=user
+      ## Github Enterprise Configuration if necessary
+      #- spring.security.oauth2.client.provider.github.authorization-uri=https://HOSTNAME/login/oauth/authorize
+      #- spring.security.oauth2.client.provider.github.token-uri=https://HOSTNAME/login/oauth/access_token
+      #- spring.security.oauth2.client.provider.github.user-info-uri=https://HOSTNAME/api/v3/user
+
```

Comment:
Remove all unnecessary lines that are not used in quick start guide.

Reply:
done